controllers/usersignup/usersignup_controller.go

```diff
@@ -700,6 +700,11 @@ func (r *Reconciler) ensureSpace(
 		return nil, false, err
 	}
 
+	err = r.updateStatusHomeSpace(ctx, userSignup, space.Name)
+	if err != nil {
+		return nil, false, err
+	}
```

Comment:
two things:
1. if the update of the UserSignup.Status fails for any reason, then the HomeSpace wouldn't be populated (Space already exists)
2. it's related to the previous item, this part of the code won't be executed for the already existing UserSignups (they already have the Space CRs created). We need to update also all existing UserSignups.

Reply:
How to update all existing UserSignups?

Reply:
I think that moving it out of the `ensureSpace` method should do the job

Reply:
Done. 9a425156d49334bd94a4da0abbd595bc3aba3a34